packages/dhis-api/src/__tests__/DhisApi/testCodesToIds.js

```diff
@@ -15,10 +15,8 @@ export const testCodesToIds = () => {
   it('should return an empty array if no codes are provided', async () => {
     const dhisApi = createDhisApi();
 
-    return Promise.all(
-      [undefined, null, []].map(codes =>
-        expect(dhisApi.codesToIds('dataElements', codes)).resolves.toStrictEqual([]),
-      ),
+    [undefined, null, []].forEach(codes =>
+      expect(dhisApi.codesToIds('dataElements', codes)).resolves.toStrictEqual([]),
```

Comment:
Contrary to the other case, this would not work here: Promises must either be awaited using `map` (like how it was done in the left), or using a sequential `for`/`for ... of` loop

If not, any failures will be thrown after the test has run, resulting in a "false positive" (tests pass while they shouldn't)

I changed that in the last commit

Reply:
I see what you mean. Thanks man.:pray: